sirepo/package_data/static/js/cloudmc.js

```diff
@@ -2600,14 +2589,14 @@ SIREPO.app.directive('jRangeSlider', function(appState, panelState) {
             fieldName: '<',
             model: '=',
             modelName: '<',
+            onChange: '&',
         },
         template: `
-            <div data-label-with-tooltip="" data-label=""></div>
             <div class="{{ sliderClass }}"></div>
             <div style="display:flex; justify-content:space-between;">
-                    <span>{{ formatFloat(field.min) }}</span>
-                    <span>{{ display(field) }}</span>
-                    <span>{{ formatFloat(field.max) }}</span>
+                <span>{{ formatFloat(field.min) }}</span>
+                <span>{{ display(field) }}</span>
+                <span>{{ formatFloat(field.max) }}</span>
             </div>
```

Comment:
In the Tokamak Reactor example, if you put the slider to Plane Position 15, it needs some formatting help:

![x](https://github.com/radiasoft/sirepo/assets/6516910/e2b2c340-ff14-41a4-ad08-1fd407ca7df1)

Reply:
Floating point numbers, man...

Reply:
[60cbb40](https://github.com/radiasoft/sirepo/pull/6813/commits/60cbb4034e24f07b47f5fd674efc69d4a464b436)

This also bolds the value to make it easier to distinguish from the limits - see how it looks to you